### DISCUSSION TITLE
Windows Support?

### DISCUSSION THREAD

**Discussion by rmorshea** [idom-team/idom#319] [General]
It's not clear whether IDOM works as expected. We need more feedback on:

- How IDOM works for Windows users
- If it's possible for new Windows contributors to set up their dev environments easily

We also need help with the following issues:

- https://github.com/idom-team/idom/issues/289 - browser-related tests on Windows don't work in our CI/CD pipeline

If you're able to help in any of the above categories, please leave a comment.

**Comment by Archmonger** [comment]
I'll test Win10 compatibility and get back with you on that.
